tests/test_tools.py

```diff
@@ -27,5 +32,114 @@ def test_ClientRedirectServer(self):
         self.assertEqual(httpd.query_params.get('code'), code)
 
 
+class _Namespace(object):
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+
+class TestRunFlow(unittest2.TestCase):
+    @mock.patch.object(sys, 'argv',
+        [sys.executable, '--noauth_local_webserver'])
```

Comment:
It really doesn't matter in this case, but I can change it to 'ignored' if you want.